telescope/selector.py

```diff
@@ -269,6 +286,15 @@ def validate_common(self, selector_dict):
         if not selector_dict.has_key('duration'):
             raise ValueError('UnsupportedDuration')
 
+        if selector_dict.has_key('client_countries'):
+            for client_country in selector_dict['client_countries']:
+                if len(client_country) != 2:
+                    raise ValueError('Requires ISO alpha-2 country code.')
+        if not selector_dict.has_key('client_countries') and \
```

Comment:
Style guide says to use parentheses for implied line continuation instead of backslash
https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Line_length#Line_length